src/lib/operators/insert.cpp

```diff
@@ -229,4 +229,8 @@ void Insert::_on_rollback_records() {
   }
 }
 
+std::shared_ptr<AbstractOperator> Insert::recreate(const std::vector<AllParameterVariant>& args) const {
+  return std::make_shared<Insert>(_target_table_name, _input_left->recreate());
```

Comment:
args should probably be passed down to _input_left->recreate()
(same might apply to the other recreate methods)

Reply:
Same in `Delete::recreate()` and `Update::recreate()`